www/js/controllers.js

```diff
@@ -194,11 +194,12 @@ angular.module('app.controllers', [])
       });
     }])
 
-  .controller('profileCtrl', ['$scope', '$stateParams', // The following is the constructor function for this page's controller. See https://docs.angularjs.org/guide/controller
+  .controller('profileCtrl', ['$scope', '$stateParams', 'Auth',// The following is the constructor function for this page's controller. See https://docs.angularjs.org/guide/controller
     // You can include any angular dependencies as parameters for this function
     // TIP: Access Route Parameters for your page via $stateParams.parameterName
-    function ($scope, $stateParams) {
-
+    function ($scope, $stateParams, Auth) {
+        Auth.user=this;
```

Comment:
`Auth.$getAuth()` will return the user. Use `console.log(Auth.$getAuth())` to see in the developer console what information is available for the user. Right now this `getDisplayName()` is returning an error since it is not defined anywhere. Remove.